src/test/resources/epub3/package-publication.feature

```diff
@@ -174,6 +174,11 @@ Feature: EPUB 3 ▸ Packages ▸ Full Publication Checks
     Then warning OPF-018 is reported
     And no other errors or warnings are reported
 
+  Scenario: Report a media overlay document with remote resources but missing the `remote-resources` property
+    When checking EPUB 'package-manifest-prop-remote-resource-overlays-error'
```

Comment:
to better align with other tests for `remote-resource` property, I suggest the test file is renamed `resources-remote-audio-in-overlays-missing-property-error` 😅